config/config.go

```diff
@@ -382,6 +382,10 @@ type ScrapeConfig struct {
 	MetricsPath string `yaml:"metrics_path,omitempty"`
 	// The URL scheme with which to fetch metrics from targets.
 	Scheme string `yaml:"scheme,omitempty"`
+	// The uncompressed response body size limit bytes for scraping targets of this config.
+	// 0 means no limit, target uncompressed response body size more than this
+	// limit will cause the scrapes to fail.
```

Comment:
To bring this in line with other doc comments and streamline it a bit, I would suggest:

An uncompressed response body larger than this many bytes will cause the scrape to fail. 0 means no limit.

Reply:
Thank your suggestion, I have add it to `docs/configuration/configuration.md`, is it right?